src/Pos/Lrc/DB/Issuers.hs

```diff
@@ -19,7 +19,7 @@ import           Pos.DB.Class      (MonadDB)
 import           Pos.DB.Error      (DBError (DBMalformed))
 import           Pos.Lrc.DB.Common (getBi, putBi)
 import           Pos.Types         (Coin, EpochIndex, EpochIndex (..), StakeholderId)
-import           Pos.Util          (maybeThrow)
+import           Pos.Util     (maybeThrow)
```

Comment:
no. I  didn't care about this formatting issues intentionally and didn't expect hate from reviewers, I supposed that it would be fixed in the future